themes/classic/modules/ps_contactinfo/nav.tpl

```diff
@@ -29,8 +29,8 @@
       {l
         s='Call us: [1]%phone%[/1]'
         sprintf=[
-          '[1]' => '<span>',
-          '[/1]' => '</span>',
+          '[1]' => "<a href='{$contact_infos['phone']}'>",
```

Comment:
it's usually a good practice to put `tel:` before the number:

```suggestion
          '[1]' => "<a href='tel:{$contact_infos['phone']}'>",
```

(except if `$contact_infos['phone']` is already formatted like this)

Reply:
Fixed !!! i forgot that